services/src/unit-test/java/uk/gov/moj/sdt/services/RetryMessageAlreadySentServiceTest.java

```diff
@@ -59,170 +60,172 @@
 import uk.gov.moj.sdt.services.utils.api.IMessagingUtility;
 import uk.gov.moj.sdt.utils.AbstractSdtUnitTestBase;
 
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.*;
+
 /**
  * Test class for the RetryMessageAlreadySentService.
  *
  * @author Manoj Kulkarni
  */
-public class RetryMessageAlreadySentServiceTest extends AbstractSdtUnitTestBase {
+@ExtendWith(MockitoExtension.class)
+class RetryMessageAlreadySentServiceTest extends AbstractSdtUnitTestBase {
     /**
      * Mocked Individual Request Dao object.
      */
+    @Mock
     private IIndividualRequestDao mockIndividualRequestDao;
 
     /**
      * The mocked ICacheable reference to the global parameters cache.
      */
+    @Mock
     private ICacheable mockCacheable;
 
     /**
      * Mocked messaging utility reference.
      */
+    @Mock
     private IMessagingUtility mockMessagingUtility;
 
     /**
      * RetryMessageAlreadySentService instance to be tested.
      */
     private RetryMessageAlreadySentService messageTaskService;
 
+    private static final String MAX_FORWARDING_ATTEMPTS = "MAX_FORWARDING_ATTEMPTS";
+    private static final String TEST_1 = "TEST_1";
+    private static final String TEST_2 = "TEST_2";
+    private static final String FORWARDED = "Forwarded";
+    private static final String FORWARDING_ATTEMPTS_ON_INDIVIDUAL_REQUEST = "Forwarding attempts on individual request";
+    private static final String TEST_COMPLETED_SUCCESSFULLY = "Test completed successfully";
+    private static final String STATUS_SET_TO_RECEIVED = "Status set to Received";
+
     /**
      * Method to do any pre-test set-up.
      */
-    @Before
+    @BeforeEach
+    @Override
     public void setUp() {
+        MockitoAnnotations.openMocks(this);

```

Comment:
This should be able to be removed as the ExtendWith(MockitoExtension.class) annotation initialises the annotated mocks in the class.

Reply:
removed

Reply:
Confirmed removal of openMocks() line.